CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 None
 
+## [v1.0.1.0](https://github.com/freckle/hspec-junit-formatter/compare/v1.0.0.4...v1.0.1.0)
+
+- Format function can be used directly without `withConfig` or `runJUnitSpec`.
+- Test case duration is now supported.
+- Failure locations are listed for some result types.
+- Timestamps in the resulting XML now display the start time of formatting.
+
```

Comment:
@pbrisbin @mjgpy3, I refreshed your review status so you can explicitly sign off.